apps/web/src/pages/Applications/ApplicationCareerTimelineEditPage/components/ExperienceEditForm.tsx

```diff
@@ -73,7 +73,16 @@ const EditExperienceForm = ({
     if (executeMutation) {
       executeMutation(args)
         .then((res) => {
-          if (res.data) {
+          if (res.error) {
+            toast.error(
+              intl.formatMessage({
+                defaultMessage: "Error: updating experience failed",
```

Comment:
Not part of the acceptance criteria, but for some of the fields that don't have any character limit, this error doesn't seem descriptive enough. Is there a reason why we don't have a word limit on the Experience Description field, for example?

## Screenshot
![Screen Shot 2023-12-20 at 09 02 24](https://github.com/GCTC-NTGC/gc-digital-talent/assets/3046459/e4a44e3b-302a-4697-a36e-24afa23303c5)